src/Client/ClientFactory.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) GitHub 2023 � Licensed as MIT.
+// Copyright (c) GitHub 2023 � Licensed as MIT.
```

Comment:
Any idea why this changed?

Reply:
So I spent about 10 minutes on it and just decided to move on... I could've down a binary check on it, but I really just wanted to move forward.

Reply:
Sounds good to me. That's interesting; I wonder if we'll see it change back in the future.